packages/core/src/property.ts

```diff
@@ -87,8 +87,32 @@ export const combine = <Properties extends readonly Property<unknown>[], Result>
 	// eslint-disable-next-line no-restricted-syntax
 	const properties: Properties = args.slice(0, args.length - 1) as never
 	const memoProject = memoMany(project)
+
 	const get = () => memoProject(...properties.map((property) => property.get()))
+
 	const doesNotEmit = properties.every((property) => property.subscribe === never.subscribe)
-	const subscribe = doesNotEmit ? never.subscribe : mergeMany(properties).subscribe
-	return newProperty(get, subscribe)
+	if (doesNotEmit) return newProperty(get, never.subscribe)
+
+	let lastValue: Result | undefined
```

Comment:
Will be conflicted with `undefined` as `Result`.

Example:
```
const a = newAtom<number | undefined>(0)

const c = combine(a, v => v)
c.subscribe()

a.set(undefined)
// no emits
```

My suggest: use empty object as initial value

Reply:
Good point

Reply:
@Fyzu I've update the code to support `undefined`. Added one more flag, just don't want to box values.